artichoke-backend/src/extn/core/time/mod.rs

```diff
@@ -21,13 +21,18 @@
 //! [`chrono-tz`]: https://crates.io/crates/chrono-tz
 
 use crate::convert::HeapAllocatedData;
+use crate::extn::prelude::*;
 
 pub mod mruby;
 pub mod trampoline;
 
 #[doc(inline)]
-pub use spinoso_time::chrono::Time;
+pub use spinoso_time::tzrs::*;
 
 impl HeapAllocatedData for Time {
     const RUBY_TYPE: &'static str = "Time";
 }
+
+pub(crate) fn convert_time_error_to_argument_error(error: TimeError) -> ArgumentError {
+    ArgumentError::from(format!("{}", error))
```

Comment:
Is there any better way of doing this?

Reply:
`impl From<tzrs::TimeError> for Error`

Then question mark will work. (Sorry for being brief, on mobile)

Reply:
Ahhh, I tried this, got some failure, but it's because I needed an `.into()` after constructing the ArgumentError - whoops.